content/en/blog/trusted-boot.md

```diff
@@ -0,0 +1,155 @@
+
+In this setup, each component is distinct: the Kernel and Initrd are separate files, whereas the root partition is a unified file system encompassing all user data (such as installed binaries, password files, sensitive information, drivers, etc.). The primary function of the initrd in this scenario is to decrypt partitions and set up mount points as it transitions control to a different partition.
+
+![kairosboot](https://github.com/kairos-io/kairos-docs/assets/2420543/b8e5d6ab-f160-43f2-adb8-395f445dbc12)
+
+
+Contrastingly, the Kairos architecture utilizes a singular image file that amalgamates the rootfs, kernel, and initrd. Here, the initrd is responsible for establishing the FS and activating the rootfs.
+
+The bootloader plays a pivotal role in this architecture by accessing image files stored within the state partition. 
+
+While the state partition remains unencrypted, the system's operational projection is derived from the encrypted user-data partition. Directories such as `/etc` and `/opt` (which can be fully customized) are overlaid, safeguarding sensitive information from being exposed within the state partition. Nevertheless, this design, along with the previously mentioned setup, is vulnerable to "Evil Maid Attacks." These attacks involve altering the system by booting from a LiveCD and modifying the booted images.
+
+
+![boot_1](https://github.com/kairos-io/kairos-docs/assets/2420543/19b61dd1-b33f-4cfe-9d0a-b60198b0cfd5)
+
+In the context of the UKI (Unified Kernel Image) architecture, which closely resembles the Kairos model, the primary distinction lies in the storage of a singular EFI image file within the boot partition. This arrangement facilitates the signing and verification of EFI files to ensure they haven't been tampered with, leveraging SecureBoot in tandem with Measured Boot. These mechanisms verify that the image is certified by a trusted authority and remains unmodified.
+
+During the boot process, cryptographic assessments are made using the Trusted Platform Module (TPM) to confirm the system's integrity. Drive decryption during boot is contingent upon consistent signing by the same authority and accurate boot measurement matching. System upgrades involve updating the boot measurement to enable access to the user data partitions.
+
+![bootingkeys](https://github.com/kairos-io/kairos-docs/assets/2420543/49ad3793-b53e-40e9-96da-58cec5d4df4c)
+
+
+
+## Security consideration
+
+The existing framework guarantees that the initial phase (UKI file) remains immutable, with only this phase capable of decrypting the drive's encrypted data. Indeed there is no pivoting into another system like in the traditional Linux boot process. This design choice ensures that the system remains a single component, with the UKI file acting as the sole point of entry for the system.
+
+Shifting to a different image introduces potential security vulnerabilities and constraints:
```

Comment:
I would change this to a "hypothetical" phrasing to avoid people reading this as if it's the case with Kairos:

```suggestion
Shifting to a different image would have introduced potential security vulnerabilities and constraints:
```

the items below need to be adjusted then.